ipfs/ipfsService.js

```diff
@@ -50,7 +50,7 @@ const get = async (key) => {
 }
 
 const stop = () => {
-  ipfs.stop(process.exit);
+  ipfs.stop();
```

Comment:
is this the bug the IPFS team fixed?

Reply:
we spoke about the bug in ipfs, that's the fix